autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/TabularColumnTransformer.py

```diff
@@ -91,4 +91,9 @@ def __call__(self, X: Union[np.ndarray, torch.tensor]) -> Union[np.ndarray, torc
         if self.preprocessor is None:
             raise ValueError("cant call {} without fitting the column transformer first."
                              .format(self.__class__.__name__))
+
+        if len(X.shape) == 1:
```

Comment:
Do you think we should support this case? having 1 feature as a column is not a very interesting case to produce targets. Many things fail downstream I think because of that.

I was actually planning to add a check to ensure at least 2D in the inputs.

Reply:
I added this because the pipeline failed during predict when inputs are single records, not if they have a single feature. So this turns e.g. a vector of length `F` into a vector of shape `[1, F]` (added batch dimension). But if you add a check that ensures this somewhere else, then I can remove this piece of code.

Reply:
Yeah I find this similar problem and was planning to fix it [here](https://github.com/automl/Auto-PyTorch/blob/d5bfd42f15a186c7e572d3187005fa52f4073a00/autoPyTorch/pipeline/components/training/data_loader/feature_data_loader.py#L14) in my new PR.

I think it makes sense to try to put all feature modification code in the same place, so debug is easy. So having it close to the dataloader/dataset is ideal. What do you think?

Reply:
Yes, thats better. Mine was just a quick fix. We should merge your PR first then, so I can rebase my code.